README.md

```diff
@@ -1,2 +1,71 @@
-# AR-based-game
-Open Projects 2021
+# AR-Based-Game
+MARS Open Projects 2021
+
+![Main Menu!](/Images/MainMenu.jpeg "MainMenu")
```

Comment:
Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

Also resize the images by assigning the dimensions and insert a title for each of them depicting what they signify. Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md